src/SFA.DAS.EmployerIncentives.Database/Tables/ApprenticeshipIncentive.sql

```diff
@@ -0,0 +1,16 @@
+﻿CREATE TABLE [dbo].ApprenticeshipIncentive
```

Comment:
The majority of this data is already stored in the IncentiveApplicationApprenticeship table, is there a reason for duplicating?

Reply:
@David-B-Read It's a different bounded context (which needs to be made clearer)